source/MRViewer/MRRibbonMenu.cpp

```diff
@@ -406,11 +406,16 @@ void RibbonMenu::drawHelpButton_()
     ImGui::PopStyleVar( 2 );
 }
 
-bool RibbonMenu::drawCustomCheckBox_( const std::vector<std::shared_ptr<Object>>& selected )
+bool RibbonMenu::drawCustomCheckBox_( const std::vector<std::shared_ptr<Object>>& selected, SelectedTypesMask selectedMask )
 {
     bool res = false;
     for ( auto& [name, custom] : customCheckBox_ )
     {
+        if ( ~custom.selectedMask & selectedMask )
```

Comment:
`(custom.selectedMask | ~selectedMask) == SelectedMaskType(-1)` ?

Reply:
it will be equivalent. the current version is more local